usecases/objects/references_add.go

```diff
@@ -57,8 +57,8 @@ func (m *Manager) AddObjectReference(ctx context.Context, principal *models.Prin
 	}
 	defer unlock()
 
-	validator := validation.New(m.vectorRepo.Exists, m.config, repl, tenantKey)
-	if err := input.validate(ctx, principal, validator, m.schemaManager); err != nil {
+	validator := validation.New(m.vectorRepo.Exists, m.config, repl)
+	if err := input.validate(ctx, principal, validator, m.schemaManager, tenantKey); err != nil {
```

Comment:
should we rename here `tenantKey` to `tenant` or `tenantName`?

Reply:
changed in renaming PR